packages/commerce-sdk-react/src/hooks/ShopperBaskets/mutation.ts

```diff
@@ -352,7 +352,8 @@ export function useShopperBasketsMutation<Action extends ShopperBasketMutationTy
                 if (
                     action === 'createBasket' ||
                     action === 'transferBasket' ||
-                    action === 'mergeBasket'
+                    action === 'mergeBasket' ||
+                    action === 'addItemToBasket'
```

Comment:
Need to add this action here to invalidate the cache, or the mini basket icon on the header won't work